gapic/api/artman_clouddebugger.yaml

```diff
@@ -0,0 +1,30 @@
+common:
+  api_name: google-clouddebugger-v2
+  proto_gen_pkg_deps:
+    - google-common-protos
+  import_proto_path:
+    - ${REPOROOT}/googleapis
+  src_proto_path:
+    - ${REPOROOT}/googleapis/google/devtools/clouddebugger/v2
+  service_yaml:
+    - ${REPOROOT}/googleapis/google/devtools/clouddebugger/clouddebugger.yaml
+  gapic_api_yaml:
+    - ${REPOROOT}/googleapis/google/devtools/clouddebugger/v2/clouddebugger_gapic.yaml
+  output_dir: ${REPOROOT}/artman/output
+  enable_batch_generation: True
+java:
+  final_repo_dir: ${REPOROOT}/google-cloud-java/google-cloud-clouddebugger
+python:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-python-clouddebugger
+go:
+  final_repo_dir: ${REPOROOT}/gapi-clouddebugger-go
+csharp:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-csharp-clouddebugger
+php:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-php-clouddebugger
+ruby:
+  final_repo_dir: ${REPOROOT}/google-cloud-ruby/google-cloud-clouddebugger
+  skip_packman: True
```

Comment:
This flag shouldn't be set unless we're integrating with a handwritten library. So if we're scaling to an unbounded amount of APIs, this flag will essentially never be set because handwritten libraries don't scale to an unbounded amount.

It seems to me that keeping a machine readable list of APIs that require this flag is doing the exact same thing as setting this flag (i.e., must set one line of code per API per language with source delivery that cares about package metadata), except that it introduces yet another input in yet another location to the generation logic.

If there's a solution that lets us integrate with handwritten libraries without having to set a configuration flag, I'm all for it.